src/Chainweb/RestAPI.hs

```diff
@@ -210,6 +213,8 @@ someChainwebServer v dbs mr =
         <> someP2pServers v (_chainwebServerPeerDbs dbs)
         <> PactAPI.somePactServers v (_chainwebServerPactDbs dbs)
         <> maybe mempty (Mining.someMiningServer v) mr
+        <> maybe mempty (someHeaderStreamServer v)
+               (bool Nothing (Just ()) hs >> _chainwebServerCutDb dbs)
```

Comment:
I wish there were better ergonomics between `Bool` and `Maybe`.

Reply:
This is definitely a code smell. `Maybe ()` is isomorphic to `Bool`, so this effectively `bool False True <some bool>`

Reply:
`_chainwebServerCutDb` gives a `Maybe`, but you're right that I can just put that in place of `Just ()`.